### DISCUSSION TITLE
使用C++推理库时不开启mkldnn识别正常，开启mkldnn后识别有问题

### DISCUSSION THREAD

**Discussion by Akumio** [PaddlePaddle/PaddleOCR#12457] [Q&A]
使用C++推理端时在未使用mkldnn时识别正确，开启rec的mdldnn后识别出现问题，请问下是哪里配置有问题呢
![微信图片_111](https://github.com/PaddlePaddle/PaddleOCR/assets/37945925/cc95538e-426b-442e-9db9-02a7e627b370)

![微信图片_222](https://github.com/PaddlePaddle/PaddleOCR/assets/37945925/c2b52519-1a88-4f6d-8d45-777bc1171c1a)

**Comment by Akumio** [comment]
测试发现使用V3模型参数未出现错误，使用V4参数时会出现以上情况

**Comment by qq61786631** [comment]
同样的，开启mkl后，速度变慢，且结果都是错误的
